modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -89,26 +91,20 @@
                  'version'       => $version,
                  'File_name'     => $fileName,
                  'File_size'     => $fileSize,
-                 'Data_dir'      => $fileBase,
-                 'uploaded_by'   => $puser,
+                 'Data_dir'      => $fullPath,
+                 'uploaded_by'   => $name,
                  'Instrument'    => $instrument,
                  'PSCID'         => $pscid,
                  'visitLabel'    => $visit,
                  'File_type'     => $fileType,
                 )
             );
-            $msg_data['newDocument']
-                = $baseURL . "/document_repository/";
+            $msg_data['newDocument'] = $baseURL .'/document_repository/';
             $msg_data['document']    = $fileName;
 
             $uploadNotifier->notify($msg_data);
 
-            $header = "Location:".
-                      " $baseURL/document_repository/?uploadSuccess=true";
-            header($header);
-
-        } else {
-            echo "There was an error uploading the file";
+            header('Location:' . $baseURL . '/document_repository/');
```

Comment:
please add `HTTP/1.1 303 See Other` before
Otherwise, This  implicitly return a 302 (see php doc http://php.net/manual/en/function.header.php and https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302)